application/browser/application.cc

```diff
@@ -193,7 +193,10 @@ void Application::SetWindowShowState<Manifest::TYPE_MANIFEST>(
 
   if (display_string == values::kDisplayModeFullscreen) {
     params->display_mode = blink::WebDisplayModeFullscreen;
-    params->state = ui::SHOW_STATE_FULLSCREEN;
+    // Set ui::SHOW_STATE_MAXIMIZED instead of ui::SHOW_STATE_FULLSCREEN
+    // because there is no way to exit from full screen mode.
+    // Refer to XWALK-4861 for more details.
+    params->state = ui::SHOW_STATE_MAXIMIZED;
```

Comment:
I know, but I couldn't find a way to add the title bar in full screen state for Windows. I should add a guard only for Windows.